docs/Imports.md

```diff
@@ -0,0 +1,96 @@
+# Imports in Exo
+
+This document provides an overview of the imports used when writing Exo.
+
+## Table of Contents
+
+1. [Standard Python Future Import](#1-standard-python-future-import)
+2. [Core Exo Module](#2-core-exo-module)
+3. [Memory Libraries](#3-memory-libraries)
+4. [Instruction Libraries](#4-instruction-libraries)
+5. [Frontend Syntax Utilities](#5-frontend-syntax-utilities)
+6. [Standard Library Modules](#6-standard-library-modules)
+   - [6.1 Scheduling Utilities](#61-scheduling-utilities)
+   - [6.2 Standard Library Functions](#62-standard-library-functions)
+7. [External Interfaces](#7-external-interfaces)
+8. [API Cursors](#8-api-cursors)
+
+
+## 1. Standard Python Future Import
+
+```python
+from __future__ import annotations
+```
+
+Enables postponed evaluation of type annotations, allowing you to use forward references in type hints without causing issues during runtime. This is necessary to support Exo's `x : f32` syntax.
+
+
+## 2. Core Exo Module
+
+```python
+from exo import *
+```
```

Comment:
It may be worth mentioning that the `from foo import bar` (or `*`) import is effectively mandatory to use things in Exo procedures, since Exo object code reserves attribute syntax (`foo.bar`) for config ops.